model_compression_toolkit/qat/keras/quantization_facade.py

```diff
@@ -60,28 +60,21 @@
 
     def qat_wrapper(n: common.BaseNode,
                     layer: Layer,
-                    qat_config: QATConfig,
-                    include_activation_quantizers: bool=False):
+                    qat_config: QATConfig):
         """
         A function which takes a computational graph node and a keras layer and perform the quantization wrapping
         Args:
             qat_config: Configuration of QAT (such as training methods for example).
             n: A node of mct graph.
             layer: A keras layer.
-            include_activation_quantizers: Whether to use the wrapper for the activation quantizer or not
-
 
         Returns: Wrapped layer
 
         """
         if _is_qat_applicable(n, DEFAULT_KERAS_INFO):
             weights_quantizers, activation_quantizers = quantization_builder(n, qat_config, DEFAULT_KERAS_INFO)
-            if include_activation_quantizers:
-                return qi.KerasQuantizationWrapper(layer, weights_quantizers, activation_quantizers)
-            else:
-                return qi.KerasQuantizationWrapper(layer, weights_quantizers)
-        else:
-            return layer
+            return qi.KerasQuantizationWrapper(layer, weights_quantizers)
```

Comment:
This change is not related to this PR right?
Is it something that was missing from the PR that added the holder to QAT?